test/rekt/broker_test.go

```diff
@@ -123,3 +123,15 @@ func TestBrokerConformance(t *testing.T) {
 	env.TestSet(ctx, t, broker.ControlPlaneConformance("default"))
 	env.TestSet(ctx, t, broker.DataPlaneConformance("default"))
 }
+
```

Comment:
remove? This was just for debugging the particular test?

Reply:
ack, it is debug. sorry!